website/content/docs/job-specification/job.mdx

```diff
@@ -105,6 +105,8 @@ job "docs" {
 - `priority` `(int: 50)` - Specifies the job priority which is used to
   prioritize scheduling and access to resources. Must be between 1 and 100
   inclusively, with a larger value corresponding to a higher priority.
+  Priority currently only has an effect when job preemption is enabled.
+  It does not have an effect on which of multiple pending jobs is run first.
```

Comment:
Where possible, we should avoid language in the docs that hints at a roadmap, because then we end up needing to keep the language in sync. (Ex. if we decide we're definitely never going to support it)

```suggestion
  Priority only has an effect when job preemption is enabled.
  It does not have an effect on which of multiple pending jobs is run first.
```

Reply:
Makes sense, thanks!